### PR TITLE
fix(docker-compose): New docker-compose version as new calling conv

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Ticket: None

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

File: testutils/infra/container_manager/docker_compose_manager.py [hunk 1]
```diff
@@ -413,7 +413,7 @@ def __init__(self, name, enterprise=False):
         super().__init__(name, extra_files)
 
     def client_services(self):
-        services = self._docker_compose_cmd("ps --service").split()
+        services = self._docker_compose_cmd("ps --services").split()
```
Comment (kacf) [inline testutils/infra/container_manager/docker_compose_manager.py:416]: On my `ubuntu:22.04` container, I get this:

```
root@f43022ecc7a6:/# docker ps --services
unknown flag: --services
See 'docker ps --help'.
root@f43022ecc7a6:/# docker service ps
"docker service ps" requires at least 1 argument.
See 'docker service ps --help'.

Usage:  docker service ps [OPTIONS] SERVICE [SERVICE...]

List the tasks of one or more services
root@f43022ecc7a6:/# 
```

So I guess both are wrong..?

Comment (oleorhagen) [reply testutils/infra/container_manager/docker_compose_manager.py:416]: Huh? I just got this from my Ubuntu 22 container...

Comment (oleorhagen) [reply testutils/infra/container_manager/docker_compose_manager.py:416]:
```
      root@6ef16a4d22ba:/# docker-compose ps --help
List containers.

Usage: ps [options] [--] [SERVICE...]

Options:
    -q, --quiet          Only display IDs
    --services           Display services
    --filter KEY=VAL     Filter services by a property
    -a, --all            Show all stopped containers (including those created by the run command)
```

Comment (oleorhagen) [reply testutils/infra/container_manager/docker_compose_manager.py:416]: Ahh, ur using the `docker` binary, not `docker-compose`

Comment (oleorhagen) [reply testutils/infra/container_manager/docker_compose_manager.py:416]: Stop confusing me more! @kacf

Comment (kacf) [reply testutils/infra/container_manager/docker_compose_manager.py:416]: Lol! Sorry, I didn't notice it either!

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
Merging, as discussed during standup

File: tests/tests/test_portforward.py [hunk 2]
```diff
@@ -136,6 +137,7 @@ def port_forward(server_url, dev_id, port_mapping, *port_mappings):
             p = subprocess.Popen(
                 [
                     "scp",
+                    "-O",
```
Comment (merlin-northern) [inline tests/tests/test_portforward.py:140]: @oleorhagen is there a `-O` option to scp?

**Pull request merged**